azkaban-common/src/main/java/azkaban/executor/Status.java

```diff
@@ -33,9 +33,8 @@
   DISABLED(100),
   QUEUED(110),
   FAILED_SUCCEEDED(120),
-  CANCELLED(130);
-  // status is TINYINT and in H2 DB the possible values are: -128 to 127
-  // so trying to store CANCELLED in H2 fails at the moment
```

Comment:
According to this doc
https://dev.mysql.com/doc/refman/5.7/en/integer-types.html
https://stackoverflow.com/questions/12812362/mysql-tinyint-as-unsigned
The signed tynyint type has the same range. 
This is confirmed by @jamiesjc's tests which use mysql.

@juhoautio how did you reach the conclusion that mysql was different?